package.json

```diff
@@ -39,7 +39,7 @@
     "electron-builder": "5.2.1",
     "electron-connect": "^0.3.7",
     "electron-packager": "^7.0.1",
-    "electron-prebuilt": "1.2.6",
+    "electron-prebuilt": "1.3.1",
```

Comment:
Nvm, I rebased and it conflicted on the version and I resolved it. Created that false bump due to an error in resolving.